firmware/src/rdm_handler.h

```diff
@@ -21,8 +21,14 @@
  * @defgroup rdm_handler RDM Request Handler
  * @brief The handler for RDM requests.
  *
- * This represents the entry point for RDM request handling. The handler
- * dispaches the request to the appropriate model.
+ * The RDM Handler manages dispatching the RDM requests to the correct model.
+ * This allows the physical device to simulate different types of RDM devices
+ * from LED drivers and moving lights to sensor-only devices and fog machines.
+ *
+ * Only a single model can be active at once, the PID_DEVICE_MODEL parameter
+ * can be used to change models at run time. Note that PID_DEVICE_MODEL and
+ * PID_DEVICE_MODEL_LIST aren't included in SUPPORTED_PARAMETERS so there are
```

Comment:
Correct, but people will be able to switch from the console anyway, which is preferred solution. Or you can use the OLA CLI.